### ISSUE TITLE
Forwarder doesn't send data after editing config store entry

### ISSUE THREAD

**Issue by jhaack** [VOLTTRON/volttron#1485]
- Start forwarder sending data to remote platform
- Observe data appearing on remote platform
- Edit config store entry for forwarder (can be blank lines)
- Data stops appearing on remote platform, but is not cached on forwarder
- 

Related to #1483

**Comment by craig8** [comment]
https://github.com/kmonson/volttron/commit/d02fd7f22db7a0128cd9870d717d72da21535776 does not fix #1485

**Comment by craig8** [comment]
Note here is a trace

````
2017-09-21 20:41:38,398 () volttron.platform.vip.agent.subsystems.rpc ERROR: unhandled exception in JSON-RPC method 'manage_get_metadata': 
````
